package.json

```diff
@@ -16,9 +16,8 @@
   "scripts": {
     "start": "NODE_ENV=production node ./src/server/server.babel.js",
     "dev": "NODE_ENV=development node ./src/server/server.babel.js",
-    "build:themes": "babel-node ./webpack/utils/buildThemeJSON.js",
-    "build": "rimraf build && npm run build:themes && concurrently \"npm run build:client\" \"npm run build:server\"",
-    "bs": "rimraf build && npm run build && npm start",
+    "build": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\"",
+    "bs": "npm run build && npm start",
     "quickstart": "npm run db:migrate && npm run build && npm run start",
     "build:client": "NODE_ENV=production webpack --config ./webpack/prod.babel.js",
     "build:client-min": "NODE_ENV=production DEPLOY=true webpack --config ./webpack/prod.babel.js",
```

Comment:
i noticed our staging server isn't set up to server the minimized version. Should we write a build:min script?